doc/api/util.md

```diff
@@ -660,18 +660,23 @@ Otherwise, returns `false`.
 See [`assert.deepStrictEqual()`][] for more information about deep strict
 equality.
 
-## util.promisify(original)
+## util.promisify(original[, options])
 <!-- YAML
 added: v8.0.0
 -->
 
 * `original` {Function}
+* `options` {Object} Optional. Can be passed to change the default behaviour.
+  Can have the following fields:
+  * `context` {any} Context to which to bind the returning function to.
```

Comment:
While it's obvious to core what context means here - I think it would be better to explain it to our users. What about:

```
`context` {any} the `this` value inside the promisified function.
```

Reply:
`the` -> `The` in @benjamingr's suggestion, but otherwise, I'm +1 on it.